Loop/Managers/AnalyticsManager.swift

```diff
@@ -118,6 +118,10 @@ final class AnalyticsManager: IdentifiableClass {
         if newValue.retrospectiveCorrectionEnabled != oldValue.retrospectiveCorrectionEnabled {
             logEvent("Retrospective correction enabled change")
```

Comment:
RC should be enabled for everyone, and allow IRC to be turned on/off.

Reply:
Would you prefer to have just the RC slider switch removed (while still keeping `retrospectiveCorrectionEnabled` so that one could disable RC by code modification), or to have `retrospectiveCorrectionEnabled` completely removed from Loop?

Reply:
`retrospectiveCorrectionEnabled` can be removed. It's still easy to remove from `enabledEffects` if someone wanted to in the code.